package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rero/rero-ils-ui",
-  "version": "0.2.1",
+  "version": "0.3.0",
```

Comment:
This has been updated on the `dev` branch. https://github.com/rero/rero-ils-ui/blob/dev/package.json#L3 Then, why it's needed here?